rfd/0001-rfd.md

```diff
@@ -0,0 +1,385 @@
+    into a separate repository. Within a directory or repository the files
+    should be named after the RFD number and title. Example:
+
+    ```bash
+    # rfd/{rfd-number}-{title}.md
+    rfd/0042-question-of-life.md
+    ```
+
+    Create the RFD file using the prototype file:
+
+    ```bash
+    cp rfd/prototypes/prototype.md rfd/0042-title.md
+    ```
+
+4.  Open a Draft Pull Request
+
+    Open a Draft Pull Request and set the title of the PR to:
+    `RFD {number}: {title}` Within the document add the pull request link to the
+    `discussion:` meta data.
+
+5.  Push your RFD branch remotely:
+
+    ```bash
+    $ git add rfd/0042-title.md
+    $ git commit -m "RFD 0042: Adding placeholder for RFD"
+    $ git push origin rfd/0042
+    ```
+
+    OPTIONAL: After your branch is pushed a table in the `rfd/README.md`
+    file will be updated. Whenever information about the state of the RFD
+    changes it updates the table as well.
+
+6.  Work on your RFD branch.
+
+    You're now ready to work on your RFD in the branch you created.
+
+    ```bash
+    $ git checkout rfd/0042
+    ```
+
+    Work on your RFD and commit your changes. Iterate on your RFD for it to be
+    ready for feedback and discussion with others. See the section on how to
+    structure an RFD for more information [link](#how-to-structure-an-rfd).
+
+7.  Ready for review
+
+    When you're ready to receive feedback on your RFD, update the state of your
+    RFD to `discussion` and push your changes remotely.
+
+    ```bash
+    $ git commit -am "RFD 0042: Add RFD for <title>"
+    $ git push origin rfd/0042
+    ```
+
+    Update your Pull Request and change its state from draft to ready for review.
+
+8.  Discuss the RFD on the Pull Request
+
+    The discussion should be had on the pull request. The RFD should be updated
+    with the feedback and changes that are made to the RFD.
+
+9.  Merge the Pull Request
+
+    When the RFD is ready and discussion , the PR can be merged into main. The
+    `state:` should be updated to `published`.
+
+    Discussion can continue on `published` RFD's. Because the discussion link
+    in the RFD `discussion:` metadata should be present to the original PR.
+    This allows for discussion on the original pull request.
+
+    If a discussion post-merge requires a larger discussion on its own, a new
+    issue should be created. In the issue title reflect the RFD number. Example:
+    "RFD 0042: Add section on X" and be sure to link back to the original PR in
+    the issue description.
+
+10. Making changes to an RFD
+
+    After an RFD has been merged, there is still the possibility to make
+    changes. Create a new pull request with the changes you like to make.
+    Similar in concept to a feature branch, each logically independent change
+    to an RFD is placed into a separate branch named after the RFD number and
+    the focus of the change (e.g. `rfd/0042_rfd_modification`). Once the branch
```

Comment:
I suggest to use kebab-case for better readability (subjective) and to be consistent with the rest of the examples

```suggestion
    the focus of the change (e.g. `rfd/0042-rfd-modification`). Once the branch
```

Reply:
Yes, I agree. I also feel it enhances the readibility